src/pip/_internal/utils/subprocess.py

```diff
@@ -51,8 +50,8 @@ def format_command_args(args):
     # has type unicode and includes a non-ascii character.  (The type
     # checker doesn't ensure the annotations are correct in all cases.)
     return ' '.join(
-        shlex_quote(str(arg)) if isinstance(arg, HiddenText)
-        else shlex_quote(arg) for arg in args
+        shlex.quote(str(arg)) if isinstance(arg, HiddenText)
+        else shlex.quote(arg) for arg in args
```

Comment:
Suggestion makes sense based on the comment. What do the maintainers think (who may have more context)?

Reply:
The suggestion looks good for me. In retrospect, we could have implemented `__unicode__` on `HiddenText` and used `six.text_type` here (it would make the intent more obvious), but that’s all history now.